client/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": "./"
```

Comment:
I'd rather avoid this because we'll likely have shared types in the future. It would probably be a good idea to use path aliases like https://github.com/MaxLeiter/maxleiter.com/blob/master/tsconfig.json#L32-L45 though.

Reply:
I agree, I can change that if you want

Reply:
That'd be great